WebDriverAgentLib/Categories/XCUIElement+FBScrolling.m

```diff
@@ -80,9 +80,9 @@ - (BOOL)fb_scrollToVisibleWithNormalizedScrollDistance:(CGFloat)normalizedScroll
   if (self.fb_isVisible) {
     return YES;
   }
-  XCElementSnapshot *scrollView = [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeScrollView];
+  XCElementSnapshot *scrollView = [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeCollectionView];
   scrollView = scrollView ?: [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeTable];
-  scrollView = scrollView ?: [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeCollectionView];
+  scrollView = scrollView ?: [self.lastSnapshot fb_parentMatchingType:XCUIElementTypeScrollView];
```

Comment:
I think any chosen order will fail in some cases. Probably best would be to find parent that is either one of those three types.